std/digest/digest.d

```diff
@@ -1021,3 +1021,53 @@ if (isDigest!T) : Digest
     ubyte[5] buf;
     assert(hash.peek(buf).toHexString() == "39A34F41");
 }
+
+/**
+ * Securely compares two digest representations. This function is $(BIGOH n)
+ * for all strings that are of equal length. Use of this function is recommended
+ * over `==` for comparing verification tokens, as it protects from brute force
+ * timing attacks that can reveal the key.
```

Comment:
Unfortunately, the Wikipedia article does not explain the specific attack in question, so I didn't use it.

I'll add a bit more explaination